pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: layout.pug
+navigationTitle: Version Support Policy
+title: Version Support Policy
+menuWeight: 5
+excerpt: Kaptain's supported version policy
+beta: false
+enterprise: false
+---
+
+D2iQ supports `N-2` of the latest `MAJOR.MINOR` release of Kaptain.
+With the GA release of Kaptain 1.0.0, support continues for Kaptain 1.0, 0.6, and 0.5, including all patch releases.
+Support for Kaptain 0.4.0 expires.
+
+Users should upgrade their Kaptain installations clusters with every new release to stay up to date with the latest features and bug fixes.
```

Comment:
My understanding is that Kaptain does not currently support in place 'upgrades'.  I.e. you must uninstall and reinstall Kaptain if you wish to use a newer version.    

If so, then I don't think we should use the word 'upgrade' here.  Instead, I'd suggest:

```suggestion
D2iQ recommends that you install the newest Kaptain version available to stay up to date with the latest features and bug fixes.
```